src/main/java/com/littleBeasts/actionMenu/AttackMenu.java

```diff
@@ -15,7 +16,7 @@
     public AttackMenu(List<CeAttack> ceAttacks) {
         super(getAttacksNames(ceAttacks));
         this.onConfirm(c -> {
-            LitiPlayer.instance().getBattle().useAttack(ceAttacks.get(c));
+            LitiBattle.getBattle().useAttack(ceAttacks.get(c));
```

Comment:
I think, renaming the method getBattle() into getCeBattle() would make it easier to read and understand

Reply:
done!